### DISCUSSION TITLE
Rollback strategy of an operator bundle

### DISCUSSION THREAD

**Discussion by Manvi1203** [redhat-openshift-ecosystem/community-operators-prod#5544] [General]
As mentioned in this [announcement ](https://github.com/redhat-openshift-ecosystem/community-operators-prod/discussions/3679), 

> Bundle Removal Restriction: Removal or update of a bundle is no longer allowed. This restriction ensures that operator users won't end up with a broken operator in case an operator is removed from an index

Can we completely remove the operator from the repository or not?

**Comment by masikor** [comment]
@Manvi1203 please read this [announcement](https://github.com/redhat-openshift-ecosystem/community-operators-prod/discussions/5031) made couple of weeks ago. When you convert your operator to FBC, you can then remove specific bundle or the whole operator.

**Reply by Manvi1203** [reply]
Is a revert similar to this permittable?
https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/4444

**Reply by masikor** [reply]
The PR you've mentioned is trying to remove specific operator bundle version in the older system - which hasn't been supported. Once you convert your operator to File Based Catalog, you then will have a full power of which bundle goes to which channel, or which one should be removed. I strongly suggest to go through the video mentioned [here](https://github.com/redhat-openshift-ecosystem/community-operators-prod/discussions/5031#discussioncomment-10496134), and then look at example PRs (the one where specific bundle would be removed is not there, but you can get the idea from similar action such as removal the whole operator, or removal from specific catalog of one ocp version). Also, it is worth to mention that once you convert your operator to FBC there is no way back to the old method. But we strongly believe it provides huge benefits so converting to FBC is recommended.
